contribute/validate-release-candidate.md

```diff
@@ -39,7 +39,7 @@ gpg --verify apache-pulsar-<release>-bin.tar.gz.asc
 
 #### Standalone service
 
-Open a terminal to start a standalone service:
+Open a terminal to start a standalone service (For 2.11.x or later, please `export PULSAR_STANDALONE_USE_ZOOKEEPER=1` before starting the service):
 
 ```shell
 bin/pulsar standalone
```

Comment:
```suggestion
PULSAR_STANDALONE_USE_ZOOKEEPER=1 bin/pulsar standalone
```

I'd prefer to update here. Even if for <2.11 (which are almost EOL) this command should work.

Reply:
updated